src/main/webapp/assets/js/reviews.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Fetches reviews from backend and displays them in sections under the appropriate business page.
+ */
+async function getReviews() {
+  const response = await fetch('/mockdatareview'); // Fetching mock data for now, to be changed
+  var reviewsList = await response.text();
+
+  /** Display reviews in sections. */
+  reviewsList = JSON.parse(reviewsList)
+  var reviewsContainer = document.getElementById('review-grid');
+  for (i=0; i<reviewsList.length; i++) {
+    var reviewSection = createReviewSection(reviewsList[i]);
+    reviewsContainer.appendChild(reviewSection);
+  }
+}
+
+/**
+ * Returns the formatted review section.
+ */
+function createReviewSection(review) {
+  const reviewWrapper = document.createElement("div");
+  reviewWrapper.class = "review-card";
```

Comment:
Regarding your question, I think this is because the correct syntax is `reviewWrapper.className = "review-card";`

Try that and see if it works! :D

Reply:
Wow thanks, it's working now :D